nixos/modules/system/boot/systemd.nix

```diff
@@ -285,6 +285,15 @@ in
       '';
     };
 
+    systemd.kbrequestUnit = mkOption {
+      default = "rescue.target";
+      type = types.str;
+      example = "rescue.target";
```

Comment:
```suggestion
```
if example repeats default, it should be omited

Reply:
```suggestion
      example = "/dev/null";
```